lib/content/package-json.hbs

```diff
@@ -40,10 +40,20 @@
   },
   "templateVersion": {{{ del }}},
   "standard": {{{ del }}},
+  "nyc": {{#if tapOver17}}{
+    "exclude": {{{ json workspaceGlobs }}}
```

Comment:
Is this related to some new change in tap 19?

Reply:
yes